portafly/src/i18n/locales/en/shared.yml

```diff
@@ -30,7 +30,7 @@ format:
   lowercase_description: Formats a text as lowercase
   format_description: Utilities to format text
 shared_elements:
-  bulk_selector: {{number}} selected
+  bulk_selector: "{{number}} selected"
```

Comment:
@porueesq @dfennessy 
When we use interpolation (variables like `{{number}}`), we have to use quotes, other way the library is unable to recognize the vars

Reply:
Understood.